### ISSUE TITLE
Release LoopBack 3.0 GA

### ISSUE THREAD

**Issue by bajtos** [strongloop/loopback#2821]
- [x] Prepare the announcement blog post
  - Just need to get @DaveYVR to publish to wordpress, confirm with @crandmck if he needs to make any more direct changes

- [x] Update loopback-workspace and tags in the npm registry to reflect the fact that 3.x is the current/latest and 2.x is a long-term-support version now. [[PR]](https://github.com/strongloop/loopback-workspace/pull/364)

  Tags to update:
  - [x] strong-remoting [[3.x]](https://github.com/strongloop/strong-remoting/pull/391) [[2.x-lts]](https://github.com/strongloop/strong-remoting/pull/392)
  - [x] loopback-datasource-juggler [[3.x]](https://github.com/strongloop/loopback-datasource-juggler/pull/1198) [[2.x-lts]](https://github.com/strongloop/loopback-datasource-juggler/pull/1199)
  - [x] loopback-connector-remote [[3.x]](https://github.com/strongloop/loopback-connector-remote/pull/69) [[1.x-lts]](https://github.com/strongloop/loopback-connector-remote/pull/70/files)
  - [x] loopback [[3.x]](https://github.com/strongloop/loopback/pull/3044) [[2.x]](https://github.com/strongloop/loopback/commit/fe1c0b605b852bae93356e0a9e1f532433a140ab)
  - [x] loopback-component-explorer [[4.x]](https://github.com/strongloop/loopback-component-explorer/pull/196) [[3.x-lts]](https://github.com/strongloop/loopback-component-explorer/commit/57efdeae3d97211ab812af519b33b6830582dbdf) [[3.x-unsupported]](https://github.com/strongloop/loopback-component-explorer/commit/9d6c3842491d92d98625445508588190e3f98394) [[2.x-lts]](https://github.com/strongloop/loopback-component-explorer/pull/197)
  - [x] workspace changes [[LB version descriptions]](https://github.com/strongloop/loopback-workspace/pull/339/files) [[explorer version]](https://github.com/strongloop/loopback-workspace/pull/364)

  Besides moving `3.x` from `next` to `latest`, we also need to move `2.x` from `latest` to `lts` because of https://github.com/npm/npm/issues/6778.

- [x] Latest/LTS Table in README for the above repos, link to LTS page on loopback.io

- [x] Release a new version from `master`:
  - [x] strong-remoting
  - [x] loopback-datasource-juggler
  - [x] loopback-connector-remote
  - [x] loopback
  - [x] loopback-component-explorer
  - [x] loopback-workspace

- [x] Finally, make the announcements, publish blog posts and make a lot of noise!
  - Official Twitter announcements will auto go out as part of @DaveYVR's blog annoucement process

- [x] Google Groups post
  - @ritch will own this and copy/paste the content to GG after the blog post goes out
  - @ritch posted at https://groups.google.com/forum/#!topic/loopbackjs/vwksf_Z6r_k

**Comment by superkhau** [comment]
tldr;

To update LB3 projects:

1. Remove publishConfig from `package.json` in master
2. Add `{"tag": "lts"}` to 2.x branch
3. Publish a new version of 3.x.x and a new version of 2.x.x

**Comment by superkhau** [comment]
Connect to https://github.com/strongloop-internal/scrum-loopback/issues/1361 for PRs related to LB3 release. @bajtos PTAL at all the PRs connected to that epic.

**Comment by superkhau** [comment]
@bajtos loopback-connector-remote doesn't have an LTS branch, I'll confirm with you tonight for how to move forward here.

**Comment by superkhau** [comment]
@bajtos I sent PRs to all those repos and linked them above. You may also use this epic to track https://github.com/strongloop-internal/scrum-loopback/issues/1361

**Comment by bajtos** [comment]
Reassigning to @ritch for the last step (post to Google Groups after the blog post was published).

**Comment by superkhau** [comment]
Checked off last items, merged in GS explorer 4.0.0 update at https://github.com/strongloop/loopback-getting-started/pull/24 and fixed LTS tags for LB3 release repos at https://github.com/strongloop-internal/scrum-loopback/issues/1373. Moving to closed column as done! :rocket:

**Comment by bajtos** [comment]
@superkhau I believe we are moving finished stories to "Done" pipeline on ZenHub instead of closing them. (I did it myself now.)

**Comment by superkhau** [comment]
@bajtos Yes we are, I'm clicking close and moving back to "Done" in ZH, but maybe it didn't reflect here.
